docs/utils/crud.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+from s3_helpers import (
+    generate_unique_bucket_name,
+)
+
+
+def create_bucket(s3_client, bucket_name):
```

Comment:
de uma revisada geral nos returns destas funcoes e se for o caso padronize em coisas parecidas, tipo o retorno da funcao do boto mesmo